tools/ckb4ibc-test/src/framework/utils/ckb.rs

```diff
@@ -305,10 +303,9 @@ pub fn fetch_ibc_connections(port: u32) -> IbcConnections {
             1,
             None,
         );
-        let cells = rt.block_on(search_connection_cell).unwrap();
+        let cells = wait_task(search_connection_cell).unwrap();
         if let Some(connection_cell) = cells.objects.into_iter().next() {
-            let tx = rt
-                .block_on(client.get_transaction(&connection_cell.out_point.tx_hash))
+            let tx = wait_task(client.get_transaction(&connection_cell.out_point.tx_hash))
```

Comment:
This is a bit inefficient (recreating the runtime when you can reuse).

Reply:
Ok I can use a shared Runtime

``` rust
fn get_rt() -> &'static Runtime {
    lazy_static::lazy_static! {
        static ref RT: Runtime = Runtime::new().unwrap();
    }
    &RT
}
```